src/app/globals.css

```diff
@@ -22,8 +22,31 @@ body {
       to bottom,
       transparent,
       rgb(var(--background-end-rgb))
-    )
-    rgb(var(--background-start-rgb));
+  ) rgb(var(--background-start-rgb));
+}
+
+@layer base {
+  h1 {
+    @apply text-3xl;
+  }
+  h2 {
+    @apply text-2xl;
+  }
```

Comment:
Test Funkcjonalnosci GitHuba, Panie Mordo. Klikam i sprawdzam co i jak, bez stresu to tylko cwiczenia.